etc/docker/elyra/Dockerfile.dev

```diff
@@ -0,0 +1,43 @@
+# syntax=docker/dockerfile:experimental
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Ubuntu 18.04 LTS - Bionic
```

Comment:
Should update this comment and image to 1.1.4. See changes made in #2273 as it required a bump to npm as well

Reply:
also do we know what the difference is between the singleuser and singleuser-example images are?

Reply:
From jupyterhub I can only find `k8s-singleuser-sample`? Where do you see the others?